packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/InputAwareWebView.java

```diff
@@ -27,8 +27,8 @@
   private View threadedInputConnectionProxyView;
   private ThreadedInputConnectionProxyAdapterView proxyAdapterView;
 
-  InputAwareWebView(Context context, View containerView) {
-    super(context);
+  InputAwareWebView(Activity activity, View containerView) {
+    super(activity);
```

Comment:
We must instantiate the WebView with the presentation context, the activity context isn't tuned for the VD size, doesn't use our custom window manager, and won't use our IMM wrapper.